pydatalab/src/pydatalab/export_utils.py

```diff
@@ -0,0 +1,169 @@
+            "@type": "Dataset",
+            "name": item.get("name", item["item_id"]),
+            "identifier": item["item_id"],
+            "dateCreated": item.get("date", datetime.now(tz=timezone.utc)).isoformat()
+            if isinstance(item.get("date"), datetime)
+            else item.get("date"),
+        }
+
+        if item.get("file_ObjectIds"):
+            files = []
+            for file_id in item["file_ObjectIds"]:
+                file_data = flask_mongo.db.files.find_one({"_id": ObjectId(file_id)})
+                if file_data:
+                    files.append({"@id": f"./{item['item_id']}/{file_data['name']}"})
+            if files:
+                item_metadata["hasPart"] = files
+
+        graph.append(item_metadata)
+
+    return metadata
+
+
+def create_eln_file(collection_id: str, output_path: str) -> None:
+    """Create a .eln file for a collection.
+
+    Parameters:
+        collection_id: ID of the collection to export
+        output_path: Path where the .eln file should be saved
+    """
+
+    collection_data = flask_mongo.db.collections.find_one({"collection_id": collection_id})
+    if not collection_data:
+        raise ValueError(f"Collection {collection_id} not found")
+
+    collection_immutable_id = collection_data["_id"]
+    child_items = list(
+        flask_mongo.db.items.find(
+            {
+                "relationships": {
+                    "$elemMatch": {"type": "collections", "immutable_id": collection_immutable_id}
+                }
+            }
+        )
+    )
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        temp_path = Path(temp_dir)
+
+        root_folder_name = collection_id
+        root_folder = temp_path / root_folder_name
+        root_folder.mkdir()
+
+        ro_crate_metadata = generate_ro_crate_metadata(collection_data, child_items)
+        with open(root_folder / "ro-crate-metadata.json", "w", encoding="utf-8") as f:
+            json.dump(ro_crate_metadata, f, indent=2, ensure_ascii=False)
+
+        for item in child_items:
+            item_folder = root_folder / item["item_id"]
+            item_folder.mkdir()
+
+            item_metadata = {k: v for k, v in item.items() if k not in ["_id", "file_ObjectIds"]}
+
+            item_metadata = _convert_objectids_in_dict(item_metadata)
+
+            with open(item_folder / "metadata.json", "w", encoding="utf-8") as f:
+                json.dump(item_metadata, f, indent=2, ensure_ascii=False)
+
+            if item.get("file_ObjectIds"):
+                for file_id in item.get("file_ObjectIds", []):
+                    file_data = flask_mongo.db.files.find_one({"_id": ObjectId(file_id)})
+                    if file_data:
+                        source_path = Path(file_data["location"])
+                        if source_path.exists():
+                            dest_file = item_folder / file_data["name"]
+                            shutil.copy2(source_path, dest_file)
```

Comment:
Somehow the actual raw data files are not making it into the `.eln` file when I try to export them